src/lib.rs

```diff
@@ -101,10 +200,14 @@ pub fn italic(input: &str) -> String {
     format!(r"\fI{}\fP", input)
 }
 
-pub fn list<C1: Troffable, C2: Troffable>(header: &'_ [C1], content: &'_ [C2]) -> String {
+pub fn list<C1: Troffable, C2: Troffable>(header: &[C1], content: &[C2]) -> String {
     format!(".TP\n{}\n{}", header.render(), content.render())
 }
 
-fn escape(input: &str) -> String {
+pub fn escape(input: &str) -> String {
     input.replace("-", r"\-")
 }
```

Comment:
Why is `escape` public?  It seems like the API should be enforcing whats escaped or not

Reply:
This was an oversight on my part, I forgot that everything is escaped when rendered, I'll fix this.